internal/transform/span.go

```diff
@@ -103,6 +103,7 @@ func (m *mapper) Transform(invocationStartedTimestamp int64) telemetry.Span {
 		isWarmStart := os.Getenv("IS_WARM_START")
 		if isWarmStart == "" && !isProvisionConcurrencyInitialization() {
 			lumigoSpan.LambdaReadiness = "cold"
+			os.Setenv("IS_WARM_START", "true")
```

Comment:
Did you check to verify that it works?
I think that the function `Transform` is called for both the start-span and end-span. If this is the case then I think that in cold start for the start-span we'll have `lumigoSpan.LambdaReadiness = "cold"` but for the end-span we'll have `lumigoSpan.LambdaReadiness = "warm"`

Reply:
you are right. fixed